DemoApp/DemoAppCoordinator.swift

```diff
@@ -141,6 +141,7 @@ private extension DemoAppCoordinator {
         let channelListVC = DemoChatChannelListVC.make(with: controller)
         channelListVC.demoRouter?.onLogout = onLogout
         channelListVC.selectedChannel = selectedChannel
+        channelListVC.components.isChatChannelListDefaultLoadingStatesEnabled = true
```

Comment:
I think this name is a bit too long and confusing. Maybe just go with `channelListLoadingStatesEnabled` ?